docs/standard/serialization/system-text-json/source-generation-modes.md

```diff
@@ -105,18 +105,22 @@ The following table shows which options in `JsonSerializerOptions` are supported
 | <xref:System.Text.Json.JsonSerializerOptions.NumberHandling>           | ❌                          |
 | <xref:System.Text.Json.JsonSerializerOptions.PropertyNamingPolicy>     | ✔️                         |
 | <xref:System.Text.Json.JsonSerializerOptions.ReferenceHandler>         | ❌                          |
+| <xref:System.Text.Json.JsonSerializerOptions.TypeInfoResolver>         |                             |
```

Comment:
I added some new options and attributes here and in the next table - if you can suggest if they're supported for source gen or not that would be great.

Reply:
They're all supported in source gen 👍